esp32-led-blink-sdk/README.md

```diff
@@ -0,0 +1,39 @@
+# esp32-led-blink-sdk
+
+This example demonstrates how to integrate with the ESP-IDF SDK via CMake and how to use the standard GPIO library to control LED from Swift. This example is specifically made for the RISC-V MCUs from ESP32 (the Xtensa MCUs are not currently supported by Swift).
+
+![Led on](pics/ledon.jpg)
```

Comment:
last nit: could you change this path to `assets/images/...`

Reply:
Sure, changed it :)